lib/spack/spack/cmd/bootstrap.py

```diff
@@ -67,24 +104,61 @@ def setup_parser(subparser):
     )
 
     list = sp.add_parser(
-        'list', help='list the methods available for bootstrapping'
+        'list', help='list all the sources of software to bootstrap Spack'
     )
     _add_scope_option(list)
 
     trust = sp.add_parser(
-        'trust', help='trust a bootstrapping method'
+        'trust', help='trust a bootstrapping source'
     )
     _add_scope_option(trust)
     trust.add_argument(
-        'name', help='name of the method to be trusted'
+        'name', help='name of the source to be trusted'
```

Comment:
I don't love the overloading of the word `source` here, but I don't have a great alternative in mind.

Reply:
Provider?

Reply:
Hmm, provider is already used for virtual dependencies.